git/git.go

```diff
@@ -307,6 +307,15 @@ func CheckoutBranch(branch string) error {
 	return run.PrepareCmd(configCmd).Run()
 }
 
+//pull changes from remote branch without version history
+func PullLatestChanges() error {
```

Comment:
This command should take stdin, stdout, stderr streams as argument so that it can connect them to the subprocess